library/plugin/reader.h

```diff
@@ -132,6 +133,38 @@ class reader
   virtual void applyCustomImporter(vtkImporter*, const std::string&) const
   {
   }
+
+  /**
+   * Set a reader option
+   * Return true if the option was found (and set), false otherwise
+   */
+  bool setReaderOption(const std::string& name, const std::string& value)
+  {
+    auto iter = this->ReaderOptions.find(name);
+    if (iter == this->ReaderOptions.end())
+    {
+      return false;
+    }
+    iter->second = value;
```

Comment:
not sure to understand the logic here. Why are you assigning the value to a local variable?

Reply:
its an iterator

Reply:
I see